mm2src/coins/eth/v2_activation.rs

```diff
@@ -114,50 +137,104 @@ pub struct EthNode {
 
 #[derive(Serialize, SerializeErrorType)]
 #[serde(tag = "error_type", content = "error_data")]
-pub enum Erc20TokenActivationError {
+pub enum EthTokenActivationError {
     InternalError(String),
     CouldNotFetchBalance(String),
+    InvalidPayload(String),
+    Transport(String),
 }
 
-impl From<AbortedError> for Erc20TokenActivationError {
-    fn from(e: AbortedError) -> Self { Erc20TokenActivationError::InternalError(e.to_string()) }
+impl From<AbortedError> for EthTokenActivationError {
+    fn from(e: AbortedError) -> Self { EthTokenActivationError::InternalError(e.to_string()) }
 }
 
-impl From<MyAddressError> for Erc20TokenActivationError {
+impl From<MyAddressError> for EthTokenActivationError {
     fn from(err: MyAddressError) -> Self { Self::InternalError(err.to_string()) }
 }
 
+impl From<GetNftInfoError> for EthTokenActivationError {
+    fn from(e: GetNftInfoError) -> Self {
+        match e {
+            GetNftInfoError::InvalidRequest(err) | GetNftInfoError::InvalidResponse(err) => {
+                EthTokenActivationError::InvalidPayload(err)
```

Comment:
Invalid response would have nothing to do with the payload correctness.

Reply:
oh, changed to transport error